lib/IAMClient.js

```diff
@@ -287,18 +285,18 @@ class VaultClient {
      * Update Quota of an account
      *
      * @param {string} accountName - account name
-     * @param {number} quota - maximum quota for the account
+     * @param {bigint} quota - maximum quota for the account
```

Comment:
* we could be more leniant, and accept either number/bigint/...
* we should also handle the bigint conversion in the callbacks : i.e. so that caller gets a BigInt, instead of having to check if this is a string/number/bigint/...